### ISSUE TITLE
megasync@4.5.3.0: hash check failed

### ISSUE THREAD

**Issue by redactedscribe** [ScoopInstaller/Extras#8277]
#### Scoop Configuration
```json
{
  "lastupdate": "2022-04-07T14:29:50.8750782+03:00",
  "SCOOP_REPO": "https://github.com/lukesampson/scoop",
  "SCOOP_BRANCH": "master",
  "aria2-enabled": true,
  "rmsshim": "kiennq",
  "shim": "kiennq",
  "aria2": false
}
```

Looks like [the same issue](https://github.com/ScoopInstaller/Extras/issues/8088) was created last month and then closed with the hash being updated manually. The problem seems to be that the manifest can't update its hash by itself and so with each time [the dl.7z](https://github.com/ScoopInstaller/Extras/blob/master/bucket/megasync.json#L14) is updated to a new version, hash errors happen.

For example, I just uninstalled MEGAsync vis Scoop and now cannot reinstall it. The latest hash for, at least, the 64-bit release should be `5e146b89bdf9b9bee36c4ea078fd3320dda27059a3dcca8bc9a3eb49b315a121`, but ideally the manifest can be fixed so that manual edits will no longer be needed.

**Comment by redactedscribe** [comment]
@issaclin32 Hello. I noticed you seem to submit commits to this repo and since I'm not sure who else to tag, could you look into this please? Scoop says "Installing 'megasync' (4.5.3.0) [64bit]" even though what is download now that a hash fix has been made is version `4.6.6.0`. The manifest still reads `"version": "4.5.3.0"`. I think these "hash check failed" issues will be necessary to submit with each new version. What gives?

**Comment by issaclin32** [comment]
The problem is caused by that *MEGASync* does not provide links for previous versions. 
As a result, there will be a (temporary) hash error whenever `https://mega.nz/MEGAsyncSetup64.exe` updates.

Our *Excavator* bot will check for update **every 4 hours**. Therefore the version/hash issue should be fixed in a short while. See [Actions](https://github.com/ScoopInstaller/Extras/actions) for details.
